dashboard/test/factories/workshop_factories.rb

```diff
@@ -340,5 +340,13 @@
       num_facilitators {2}
       each_session_hours {7}
     end
+
+    factory :build_your_own do
```

Comment:
nit: would it be clearer to name this something like `build_your_own_workshop`? I don't feel that strongly though if you think this name is better!

Reply:
Removing this entirely!